src/algebra/ordered_monoid.lean

```diff
@@ -429,34 +236,27 @@ instance [lattice α] : lattice (with_zero α) := with_bot.lattice
 
 instance [linear_order α] : linear_order (with_zero α) := with_bot.linear_order
 
-lemma mul_le_mul_left {α : Type u}
-  [ordered_comm_monoid α] :
+local attribute [instance] with_zero.has_le
```

Comment:
This looks like a bad idea: the lemmas below will now use your local instance instead of the global instance.

Reply:
Ok, I removed this line and it seems that, at least this file, compiles with no errors!